firmware.py

```diff
@@ -5,13 +5,55 @@
 
 app=Flask(__name__)
 app.config.from_object(__name__)
+app.config.update(dict(
+    DATABASE = os.path.join(app.root_path, 'firmware.db'),
+    SECRET_KEY='such secret much wow so key no guess muh'
+))
+
+
+def connect_db():
+    rv=sqlite3.connect(app.config['DATABASE'])
```

Comment:
Please add spaces before and after an operator. There is one exception in a case like this: `call_a_function(12, key_word_param='a value')`
Please do this everywhere